wiki/fr/fr-Getting-Started.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: wiki
+title: "Pour commencer"
+lang: "fr"
+permalink: "/wiki/Getting-Started"
+---
+
+# Commencer à utiliser Jamulus
+
+## Vous avez besoin de
+
+1. **Un ordinateur** au moins sous **Windows 10**, **MacOS X** El Capitan (v10.11) ou plus réçent, ou **GNU/Linux**.
+1. **Une connection Internet filaire** (n'utilisez pas la Wi-Fi, cela va causer des problèmes de son). La plupart des connexions standard suffisent[^1].
+1. **Des microphones filaires - pas de haut parleurs** pour éviter de l'écho ou des larsen et pour minimiser le délai[^2].
+
+## Maximiser la qualité et diminuer le délai
+
+### 1. Utilisez une interface audio∕un microphone externe, pas votre carte son interne
+
+Sauf dans le cas où vous êtes un musicien ayant un microphone (USB), il est fortement recommandé d'utiliser une carte son : un appareil sur lequel vous pouvez brancher votre instrument. Celle-çi se connectera à votre PC, généralement par USB. Visitez [la section matériel](Hardware-Setup).
+
+### 2. Réduisez votre trafic Internet et fermez les programmes de votre PC
+
+Assurez-vous que Jamulus ait le maximum de ressources sur votre machine ainsi que via votre connexion réseau quand vous l'utilisez.
```

Comment:
I said that :
> Essayez de faire en sorte que rien sur votre machine ou votre réseau ne vienne concurrencer Jamulus lorsque vous l'utilisez.

We could merge with something like that : 
« Assurez vous que Jamulus ait le maximum de ressources disponibles sur votre machine ainsi que de bande passante lorsque vous l'utilisez. Faites en sorte qu'aucun autre programme vienne concurrencer Jamulus losque vous l'utiliser et qu'aucun autre utilisateur sur votre réseau ne consomme trop de bande passante ».

_Jamulus is not for peace in the home!_ :D (small aside, not to be included in the doc.)

Reply:
I commited "Essayez de vous assurer que rien sur votre machine ou votre réseau ne soit en concurrence avec Jamulus lorsque vous l'utilisez." but i prefer "ne risque de gêner Jamulus lorsque vous l'utiliser"

Reply:
@Saimyx  say
> I commited "Essayez de vous assurer que rien sur votre machine ou votre réseau ne soit en concurrence avec Jamulus lorsque vous l'utilisez." but i prefer "ne risque de gêner Jamulus lorsque vous l'utiliser"

I've used _que rien sur votre machine n'entre en concurrence_ but not happy with _concurrence_ too. However it's better than _gêner_ (IMHO).